app/src/main/java/uk/gov/onelogin/login/ui/biooptin/BiometricsOptInScreen.kt

```diff
@@ -30,14 +30,11 @@ import uk.gov.android.ui.theme.GdsTheme
 import uk.gov.android.ui.theme.hintTextGrey
 import uk.gov.android.ui.theme.m3.Typography
 import uk.gov.android.ui.theme.smallPadding
-import uk.gov.onelogin.login.biooptin.BiometricPreference
 
 @Suppress("LongMethod")
 @Composable
 fun BiometricsOptInScreen(
```

Comment:
It's definitely a nice refactor when responsibility is moved and the parameters start disappearing